server/app/views/admin/TranslationFormView.java

```diff
@@ -53,7 +53,7 @@ protected final DivTag renderLanguageLinks(long entityId, Locale currentlySelect
    * Given the ID of the entity to translate and a locale for translation, returns a link
    * destination URL for the edit form to translate the entity in the given locale.
    */
-  protected abstract String languageLinkDestination(long entityId, Locale locale);
+  protected abstract String languageLinkDestination(String entityId, Locale locale);
```

Comment:
(blocking) Can you update entityId to be entityName along with the javadoc

Reply:
Done, good call